clippy_lints/src/collection_is_never_read.rs

```diff
@@ -0,0 +1,125 @@
+use clippy_utils::diagnostics::span_lint;
+use clippy_utils::get_enclosing_block;
+use clippy_utils::get_parent_node;
+use clippy_utils::path_to_local_id;
+use clippy_utils::ty::is_type_diagnostic_item;
+use clippy_utils::visitors::for_each_expr_with_closures;
+use core::ops::ControlFlow;
+use rustc_hir::Block;
+use rustc_hir::ExprKind;
+use rustc_hir::HirId;
+use rustc_hir::Local;
+use rustc_hir::Node;
+use rustc_hir::PatKind;
```

Comment:
```suggestion
use rustc_hir::{Block, ExprKind, HirId, Local, Node, PatKind};
```
You can combine multiple imports (also in other places) to reduce the code a bit.

Reply:
Done.